### DISCUSSION TITLE
App translation

### DISCUSSION THREAD

**Discussion by only1avetrill** [DevToys-app/DevToys#465] [Ideas]
After downloading the program, I realized that it did not have a very correct and incomplete translation into Russian. I would like to help and translate this program into Russian completely. If needed, please let me know.

**Comment by veler** [comment]
Hi,
Sure, feel free to contribute to the translation.
You can find all the Russian (translated and non-translated yet) resources here: https://github.com/veler/DevToys/tree/main/src/dev/impl/DevToys/Strings/ru-RU

Additionally, here are some instructions at how to build the project so you can test your changes: https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md#how-to-build-and-run-devtoys-from-source
